h2o-py/h2o/estimators/aggregator.py

```diff
@@ -19,22 +19,64 @@ class H2OAggregatorEstimator(H2OEstimator):
     """
 
     algo = "aggregator"
-    param_names = {"model_id", "training_frame", "response_column", "ignored_columns", "ignore_const_cols",
-                   "target_num_exemplars", "rel_tol_num_exemplars", "transform", "categorical_encoding",
-                   "save_mapping_frame", "num_iteration_without_new_exemplar", "export_checkpoints_dir"}
 
-    def __init__(self, **kwargs):
+    def __init__(self,
+                 model_id=None,  # type: Optional[H2OEstimator]
+                 training_frame=None,  # type: Optional[H2OFrame]
+                 response_column=None,  # type: Optional[str]
+                 ignored_columns=None,  # type: Optional[List[str]]
+                 ignore_const_cols=True,  # type: bool
+                 target_num_exemplars=5000,  # type: int
+                 rel_tol_num_exemplars=0.5,  # type: float
+                 transform="normalize",  # type: Literal["none", "standardize", "normalize", "demean", "descale"]
+                 categorical_encoding="auto",  # type: Literal["auto", "enum", "one_hot_internal", "one_hot_explicit", "binary", "eigen", "label_encoder", "sort_by_response", "enum_limited"]
+                 save_mapping_frame=False,  # type: bool
+                 num_iteration_without_new_exemplar=500,  # type: int
+                 export_checkpoints_dir=None,  # type: Optional[str]
+                 ):
+        """
+        :param model_id: Destination id for this model; auto-generated if not specified. (default:None).
```

Comment:
I used parenthesis because I had  issues with punctuation: some help string ends with dots, some other don't and I didn't want a hacky logic like checking if the sentences ends with a dot or whatever.
I found a workaround though  and fixed our `wrap` function to handle multilines, so now I can sefely add the defaults as the last line.